app/components/Link/Link.module.scss

```
@@ -0,0 +1,16 @@
+.link {
+  cursor: pointer;
+
+  font-size: inherit;
+  line-height: inherit;
+  text-decoration: underline;
+
+  & :hover {
+    opacity: 0.7;
+  }
+  & :focus {
```

```suggestion
  &:hover {
    opacity: 0.7;
  }
  &:focus {
```
pseudo styles weren't working